client/shipping-label/state/reducer.js

```diff
@@ -422,21 +422,24 @@ reducers[ SET_PACKAGE_TYPE ] = ( state, { packageId, boxTypeId } ) => {
 	const newPackages = { ...state.form.packages.selected };
 	const oldPackage = newPackages[ packageId ];
 
+	const oldBox = state.form.packages.all[ oldPackage.box_id ];
+	const newBox = state.form.packages.all[ boxTypeId ];
+	const weight =
```

Comment:
@dmsnell It may not be actual bug (i.e. following the standards), but it's certainly not expected.

> also, I'm guessing that the kinds of errors we will get from floating-point oddities will be far lower than what would be significant with the weight of any shippable thing

Yeh, that's the assumption we need to double-check here.